reports/namespace-usage/go.mod

```diff
@@ -0,0 +1,48 @@
+module github.com/ministryofjustice/cloud-platform-how-out-of-date-are-we/reports/namespace-usage
+
+go 1.17
+
+require (
+	k8s.io/api v0.23.3
+	k8s.io/apimachinery v0.23.3
+	k8s.io/metrics v0.23.3
+)
+
+require (
+	github.com/aws/aws-sdk-go v1.42.40 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/go-logr/logr v1.2.0 // indirect
+	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/golang/protobuf v1.5.2 // indirect
+	github.com/google/go-cmp v0.5.6 // indirect
+	github.com/google/go-github v17.0.0+incompatible // indirect
+	github.com/google/go-querystring v1.1.0 // indirect
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/googleapis/gnostic v0.5.5 // indirect
+	github.com/imdario/mergo v0.3.12 // indirect
+	github.com/jmespath/go-jmespath v0.4.0 // indirect
+	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/ministryofjustice/cloud-platform-environments v1.1.4-0.20220203124601-5370c388698b // indirect
```

Comment:
I'm confused; there isn't a 1.1.4
https://github.com/ministryofjustice/cloud-platform-environments/tags

Reply:
I removed the hard dependency of getting v1.1.3 and did `go get github.com/ministryofjustice/cloud-platform-environments`. That added an indirect version tag. I think that way i can get the latest from the main branch of env package. DO you prefer having the env package tagged and use the tag.